Formula/flake8.rb

```diff
@@ -30,15 +30,33 @@ class Flake8 < Formula
     sha256 "379358498f58f69157b53f59f46aefda0e9a3eb81365238f69fbedf7014e21ab"
   end
 
+  resource "configparser" do
+    url "https://files.pythonhosted.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz"
+    sha256 "5308b47021bc2340965c371f0f058cc6971a04502638d4244225c49d80db273a"
+  end
+
+  resource "enum34" do
+    url "https://files.pythonhosted.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz"
+    sha256 "8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1"
+  end
+
   def install
     ENV.prepend_create_path "PYTHONPATH", "#{libexec}/vendor/lib/python2.7/site-packages"
 
-    %w[pycodestyle pyflakes mccabe].each do |dependency|
+    %w[pycodestyle pyflakes mccabe enum34].each do |dependency|
       resource(dependency).stage do
         system "python", *Language::Python.setup_install_args(libexec/"vendor")
       end
     end
 
+    resource("configparser").stage do
+      # NOTE(sigmavirus24): configparser uses namespace packages which do not
+      # play well with --single-version-externally-managed
```

Comment:
So to add a bit more context, I tried this out at first by having configparser as part of the array above and kept seeing import errors. Only by breaking it out did it work. I think this is because brew (correctly) installs dependencies in a subdirectory by convention. Removing `--single-version-externally-managed` made this work better. I have no good way of explaining it =/ I barely understand this myself.